recipes/cutde/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cutde" %}
+{% set version = "21.8.20" %}
+{% set sha = "b804249cf138946c110abb7bb93e5557ae457c45eb66d1e0660ecdd5664aec94" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cutde-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<37]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - llvm-openmp  # [osx]
+    - _openmp_mutex  # [not osx]
+    - pybind11
+    - mako
+
+  host:
+    - python
+    - numpy
+    - pip
+    - llvm-openmp  # [osx]
+    - _openmp_mutex  # [not osx]
+    - pybind11
+    - mako
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - pyproj
+    - llvm-openmp  # [osx]
+    - _openmp_mutex  # [not osx]
```

Comment:
These should be added implicitly here.
```suggestion
```

Reply:
When I left them out, I got an overlinking error saying that I needed to include them in requirements/run. Otherwise, I wouldn't have the openmp shared libraries available at runtime.

Reply:
Nevermind, when combined with your other suggestions, this works well. =)